src/assets/style/_mixins.scss

```diff
@@ -0,0 +1,17 @@
+//Mobile-first Media Queries
+
+//Breakpoints
+$tablet: (min-width: 768px);
+$desktop: (min-width: 992px);
+
+@mixin tablet {
```

Comment:
can you please make sure you rename this mixin to be more specific. example - @mixin media-breakpoint-tablet() 
Same for desktop - @mixin media-breakpoint-desktop()

Reply:
as our variables would just be px values, can we have mixin as - 

@media (min-width: $media-breakpoint-tablet) {
        @content
}